Blockzilla/AppDelegate.swift

```diff
@@ -409,6 +412,55 @@ extension AppDelegate {
         }
 
         Glean.shared.initialize(uploadEnabled: Settings.getToggle(.sendAnonymousUsageData))
+
+        // Proof-of-Concept application-services guff here.
+        //
+        // We're just seeing if we can successfully import and use various application-services
+        // components, they don't really do anything in the app yet.
+        //
+        
+        // Hook up basic logging.
+        if !RustLog.shared.tryEnable({ (level, tag, message) -> Bool in
+            let logString = "[RUST][\(tag ?? "no-tag")] \(message)"
+            print(logString)
+            return true
+        }) {
+            print("ERROR: Unable to enable logging from Rust")
+        }
+
+        // Enable networking.
+        Viaduct.shared.useReqwestBackend()
+
+        var nimbusDbPath: String {
+            let profilePath = FileManager.default.containerURL(
+                forSecurityApplicationGroupIdentifier: AppInfo.sharedContainerIdentifier
+            )?
+            .appendingPathComponent("profile.profile")
+            .path
+            let dbPath = profilePath.flatMap {
+                URL(fileURLWithPath: $0).appendingPathComponent("nimbus.db").path
+            } ?? ""
+            
+            return dbPath
+        }
+
+        let errorReporter: NimbusErrorReporter = { err in
+            print(err)
+        }
+        do {
+            let nimbus = try Nimbus.create(
+                NimbusServerSettings(url: URL(string: "https://firefox.settings.services.mozilla.com")!),
```

Comment:
Is it OK to ship with a hardcoded URL for Nimbus? We have #1993 to move this endpoint into a CI configuration. That can be a followup patch that the Focus team does.

Reply:
This is mainly a test code to help validate that the swift packages are actually working and we can actually fetch APIs from the xcframework. Definitely intended (and all the code I added in AppDelegate.swift) to be deleted and replaced with the "production" code.